### DISCUSSION TITLE
where has corona virus info gone to....?

### DISCUSSION THREAD

**Discussion by legraml** [matt8707/hass-config#70] [Q&A]
I recently updated button_card_template.yaml and noticed that the corona virus ticker has gone? Or has it been moved to another place?

**Comment by matt8707** [comment]
I removed it here https://github.com/matt8707/hass-config/commit/b5fd84b7310d60ccd2710f1e3cff9957f4f6d6a3

History https://github.com/matt8707/hass-config/tree/c560bded8288bcfbb15b1c019475efa8d81de3bf
